SplunkRumWorkspace/SplunkRum/SplunkRum/NetworkType.swift

```diff
@@ -44,12 +44,6 @@ private func setConnectionType(_ type: String?) {
     }
     currentNetInfo.hostConnectionType = type
 
-    if type != "cell" {
-        currentNetInfo.hostConnectionSubType = nil
-        currentNetInfo.carrierCountryCode = nil
-        currentNetInfo.carrierNetworkCode = nil
-        currentNetInfo.carrierIsoCountryCode = nil
-    }
```

Comment:
Shouldn't this part stay though? `setCarrierInfo` is only called from `serviceSubscriberCellularProvidersDidUpdateNotifier` callback, is this callback invoked when the connection type changes?